src/style.css

```diff
@@ -1 +1,105 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 600;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the universal selector '*' for setting box-sizing as it impacts performance. Instead, set box-sizing on specific elements that require it.